CMakeLists.txt

```diff
@@ -182,16 +182,23 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ${ENABLE_SHARED})
 
 find_package(Backtrace)
 
+find_package(urcu REQUIRED)
```

Comment:
@dlambrig looks like we have an unconditional find_package(urcu), and again in if(LINUX);  only the LINUX block sets HAVE_URCU;  is the LINUX check needed now?